### PR TITLE
Offline and save project pages

### PR DESCRIPTION
**Pull request by ganicke**
- [x] Bring in _offline.md_ changes from ``v0``.
- [x] Add a "Saving Projects" page to describe where projects are stored.

Need to include a mention of packages and if/where they are kept when added?

### CODE REVIEW

File: common-docs/SUMMARY.md [hunk 1]
```diff
@@ -67,6 +67,7 @@
     * [Translate](/translate)
     * [Sharing projects](/share)
     * [Offline support](/offline)
+    * [Saving projects](/save-project)
```
Comment (pelikhan) [inline common-docs/SUMMARY.md]: Maybe just “save”

**Comment by pelikhan** [comment]
Merge https://github.com/Microsoft/pxt/pull/4363 in this one too?

File: common-docs/SUMMARY.md [hunk 2]
```diff
@@ -67,7 +67,7 @@
     * [Translate](/translate)
     * [Sharing projects](/share)
     * [Offline support](/offline)
-    * [Saving projects](/save-project)
+    * [Save](/save-project)
```
Comment (pelikhan) [inline common-docs/SUMMARY.md]: can we rename the page "save"?

File: common-docs/faq.md [hunk 3]
```diff
@@ -4,11 +4,27 @@
...
+
+## Can I use MakeCode when I'm not connected to the internet?
+
+Yes, when MakeCode first loads in the browser, it tells the browser to download the files it needs to work locally. If you later want to add a package to use with your program, you will need to connect again. See the page about using MakeCode [offline](/offline).
+
+## What happens when I clear the browser's cache and temporary files?
+
+When you clear browsing data, all of the saved projects and files that support MakeCode working offline are removed. Any project that you've shared to the public cloud is still there.
+
+## Where does a project that I share go?
```
Comment (pelikhan) [inline common-docs/faq.md]: we should add an identifer to each question so that we can also override them

**Comment by ganicke** [comment]
I added the changes from #4363 into the _offline.md_ in here.

I'm ignorant of any OneNote integration as mentioned by @Jaqster in the sharing comment: "What's the best way for students to submit their work?" Is there an export from shared projects for that?

You want a section on "files". You mean talk about using Explorer and what the files are?

**Comment by pelikhan** [comment]
Clicking on the "save" button is the most reliable way of getting a copy of the code. The browser cache can be destroyed in various (unintented ways): browser update, clearing all cookies, school policy that erases profiles after logout ,etc...

**Comment by pelikhan** [comment]
Students can share their work in multiple ways:

- use the share button and send the generated URL to the teacher. URLs copy-pasted into OneNote (or any https://embed.ly compatible web sites) will automatically be expanded into a read-only editor.
- save the file and upload it into the school/classroom web site
- go to javascript and copy/paste the Javascript code. (This approach does not work if you are using Extensions).

**Comment by ganicke** [comment]
BTW, ``## Miscellaneous`` isn't not pulling into the targets' SUMMARYs. How discoverable then are these pages?

**Comment by pelikhan** [comment]
FAQ should be bumped up to the top.

**Comment by pelikhan** [comment]
They are not discoverable enough right now...

**Pull request merged**